packages/orbit-components/src/Wizard/Wizard.stories.tsx

```diff
@@ -1,156 +1,140 @@
 import * as React from "react";
 import { action } from "@storybook/addon-actions";
+import type { Meta, StoryObj } from "@storybook/react";
 
 import Stack from "../Stack";
 import RenderInRtl from "../utils/rtl/RenderInRtl";
 
 import Wizard, { WizardStep } from ".";
 
-export default {
+type WizardPropsAndCustomArgs = React.ComponentProps<typeof Wizard> & {
+  steps: string[];
+};
+
+const meta: Meta<WizardPropsAndCustomArgs> = {
   title: "Wizard",
+  component: Wizard,
+
+  parameters: {
+    controls: {
+      exclude: ["steps", "onChangeStep", "direction", "lockScrolling", "labelClose"],
+    },
+  },
+
+  args: {
+    id: "wizard",
+    completedSteps: 2,
+    steps: [
+      "Search",
+      "Passenger details",
+      "Ticket fare",
+      "Customize your trip",
+      "Kiwi.com guarantee",
+      "Seating",
+      "Overview & payment",
+    ],
+    onChangeStep: action("onChangeStep"),
+  },
+
+  argTypes: {
+    direction: {
+      options: ["row", "column"],
+      control: {
+        type: "select",
+      },
+    },
+    completedSteps: {
+      control: {
+        type: "range",
+        min: 0,
+        max: 7,
+        step: 1,
+      },
+    },
+  },
 };
 
-export const Default = ({ direction, labelClose, labelProgress }) => {
-  return (
-    <Wizard
-      id="wizard"
-      direction={direction}
-      completedSteps={2}
-      activeStep={1}
-      labelClose={labelClose}
-      labelProgress={`2 ${labelProgress} 6`}
-      onChangeStep={action("onChangeStep")}
-    >
-      <WizardStep title="Search" />
-      <WizardStep title="Passenger details" />
-      <WizardStep title="Ticket fare" />
-      <WizardStep title="Kiwi.com guarantee" isCompleted />
-      <WizardStep title="Seating" />
-      <WizardStep title="Overview & Payment" />
+export default meta;
+type Story = StoryObj<WizardPropsAndCustomArgs>;
+
+export const Default: Story = {
+  render: ({ completedSteps, steps, ...args }) => (
+    <Wizard {...args} activeStep={completedSteps} completedSteps={completedSteps}>
```

Comment:
I excluded `activeStep` from controls/args as before we have been setting argType for `activeStep` dynamically based on the value of `completedSteps` and `steps`. Unfortunately, this is not possible with CSF3 (and it seems to me that it's not working currently on master either 🤔).

I also hardcoded `steps` because we have also been setting value of `completedSteps` dynamically based on the length of `steps`. With hardcoded value of steps, we can just set argType of `completedSteps`. And then we can calculate value of `activeStep` which can be `<= completedSteps` but I think it's fine to always have `= completedSteps`.